digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/kmtk/KMTKClient.scala

```diff
@@ -0,0 +1,595 @@
+package fi.liikennevirasto.digiroad2.client.kmtk
+
+import java.io.{IOException, InputStream}
+import java.net.URLEncoder
+
+import fi.liikennevirasto.digiroad2.asset._
+import fi.liikennevirasto.digiroad2.client.vvh._
+import fi.liikennevirasto.digiroad2.linearasset.RoadLinkLike
+import fi.liikennevirasto.digiroad2.util.KMTKAuthPropertyReader
+import fi.liikennevirasto.digiroad2.util.LogUtils.time
+import fi.liikennevirasto.digiroad2.{Point, PointM}
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.impl.client.{CloseableHttpClient, HttpClientBuilder}
+import org.joda.time.DateTime
+import org.joda.time.format.{DateTimeFormat, DateTimeFormatter}
+import org.json4s.jackson.JsonMethods.parse
+import org.json4s.{DefaultFormats, Formats, StreamInput}
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+case class KMTKID(uuid: String, version: Long)
+
+case class KMTKGeometry(/*`type`: String, */ coordinates: Seq[Seq[Double]]) {
```

Comment:
KMTK-interface had a bug: there were duplicate type-field, which caused error while parsing the json.